.travis.yml

```diff
@@ -13,13 +13,13 @@ before_install:
 install:
   - sudo apt-get install bazel
   - go get -u github.com/alecthomas/gometalinter
-  - go get -u github.com/bazelbuild/buildifier/buildifier
+  - go get -u github.com/bazelbuild/buildtools/buildifier
   - gometalinter --install
   - go install ./...
 
 script:
   - verify/verify-boilerplate.sh --rootdir="$GOPATH/src/k8s.io/repo-infra" -v
   - verify/verify-go-src.sh --rootdir "$GOPATH/src/k8s.io/repo-infra" -v
-  - buildifier -mode=check $(find . -name BUILD -o -name '*.bzl' -type f)
+  - buildifier -mode=check $(find . -name BUILD -o -name '*.bzl' -type f -not -wholename '*/vendor/*')
```

Comment:
why are we skipping vendor?

Reply:
Caught the following error when building with vendor (https://travis-ci.org/kubernetes/repo-infra/builds/320873011):
```
$ buildifier -mode=check $(find . -name BUILD -o -name '*.bzl' -type f)
./vendor/github.com/bazelbuild/buildtools/build/build_defs.bzl # reformat 
The command "buildifier -mode=check $(find . -name BUILD -o -name '*.bzl' -type f)" exited with 4.
```

Reply:
Ha, that's amazing.

(I will maybe go send them a PR to fix that. :)